README.md

```diff
@@ -1,97 +1,44 @@
 
-> A library that helps Helia users pin content via the IPFS pinning-services-api-spec
-
-## Table of contents <!-- omit in toc -->
-
-- [Install](#install)
-  - [Browser `<script>` tag](#browser-script-tag)
-- [Documentation](#documentation)
-  - [Create remote pinner](#create-remote-pinner)
-  - [Add a pin](#add-a-pin)
-  - [Replace a pin](#replace-a-pin)
-- [Lead Maintainer](#lead-maintainer)
-- [Contributing](#contributing)
-- [License](#license)
-- [Contribute](#contribute)
-
-## Install
-
-```console
-$ npm i @helia/remote-pinning
-```
-
-### Browser `<script>` tag
-
-Loading this module through a script tag will make it's exports available as `HeliaRemotePinning` in the global namespace.
-
-```html
-<script src="https://unpkg.com/@helia/remote-pinning/dist/index.min.js"></script>
-```
-
-## Documentation
-
-### Create remote pinner
-
-```typescript
-import { unixfs } from '@helia/unixfs'
-import { Configuration, RemotePinningServiceClient } from '@ipfs-shipyard/pinning-service-client'
-import { createHelia } from 'helia'
-import { createRemotePinner } from '@helia/remote-pinning'
-
-const helia = await createHelia()
-const pinServiceConfig = new Configuration({
-  endpointUrl: `${endpointUrl}`, // the URI for your pinning provider, e.g. `http://localhost:3000`
-  accessToken: `${accessToken}` // the secret token/key given to you by your pinning provider
-})
-
-const remotePinningClient = new RemotePinningServiceClient(pinServiceConfig)
-const remotePinner = createRemotePinner(helia, remotePinningClient)
-```
-
-### Add a pin
+> A IPFS Pinning Service API implementation powered by Helia
 
-```typescript
-const heliaFs = unixfs(helia)
-const cid = await heliaFs.addBytes(encoder.encode('hello world'))
-const addPinResult = await remotePinner.addPin({
-  cid,
-  name: 'helloWorld'
-})
-```
+## About
 
-### Replace a pin
+This repo contains a server implementation of the IPFS [Delegated Routing V1 HTTP API](https://specs.ipfs.tech/routing/http-routing-v1/) along with a client that can be used to interact with any compliant server implementation.
 
-```typescript
-const newCid = await heliaFs.addBytes(encoder.encode('hi galaxy'))
-const replacePinResult = await remotePinner.replacePin({
-  newCid,
-  name: 'hiGalaxy',
-  requestid: addPinResult.requestid
-})
-```
+# Packages
 
-## Lead Maintainer
+- [`packages/client`](https://github.com/ipfs/helia-remote-pinning/tree/main/packages/client) Add remote pinning capabilities to Helia
+- [`packages/interop`](https://github.com/ipfs/helia-remote-pinning/tree/main/packages/interop) Interop tests for the IPFS Pinning Service API client and server powered by Helia
+- [`packages/server`](https://github.com/ipfs/helia-remote-pinning/tree/main/packages/server) An IPFS Pinning Service API server powered by Helia
 
-[SgtPooki](https://github.com/sgtpooki)
+> The Delegated Routing V1 HTTP API powered by Helia
```

Comment:
not sure where the delegated routing stuff is coming from

```suggestion
> The Delegated Routing V1 HTTP API powered by Helia
```

Reply:
I copy/pasted a lot of the repo structure..

Reply:
Updated